spec/JSON.Parser.Spec.savi

```diff
@@ -23,36 +23,36 @@
 
   :it "parses numbers"
     assert: @example("123", <<<
-      [JSON.TokenNumberPre, 0, 0, 0, 0.0, ""]
-      [JSON.TokenNumber, 0, 3, 123, 0.0, ""]
+      [JSON.Token.NumberPre, 0, 0, 0, 0.0, ""]
+      [JSON.Token.Number, 0, 3, 123, 0.0, ""]
     >>>)
     assert: @example("-123", <<<
-      [JSON.TokenNumberPre, 0, 0, 0, 0.0, ""]
-      [JSON.TokenNumber, 0, 4, -123, 0.0, ""]
+      [JSON.Token.NumberPre, 0, 0, 0, 0.0, ""]
+      [JSON.Token.Number, 0, 4, -123, 0.0, ""]
     >>>)
     assert: @example("123.456", <<<
-      [JSON.TokenNumberPre, 0, 0, 0, 0.0, ""]
-      [JSON.TokenNumber, 0, 7, 0, 123.456, ""]
+      [JSON.Token.NumberPre, 0, 0, 0, 0.0, ""]
+      [JSON.Token.Number, 0, 7, 0, 123.456, ""]
     >>>)
     assert: @example("-123.456", <<<
-      [JSON.TokenNumberPre, 0, 0, 0, 0.0, ""]
-      [JSON.TokenNumber, 0, 8, 0, -123.456, ""]
+      [JSON.Token.NumberPre, 0, 0, 0, 0.0, ""]
+      [JSON.Token.Number, 0, 8, 0, -123.456, ""]
     >>>)
     assert: @example("123e2", <<<
-      [JSON.TokenNumberPre, 0, 0, 0, 0.0, ""]
-      [JSON.TokenNumber, 0, 5, 0, 12300, ""]
+      [JSON.Token.NumberPre, 0, 0, 0, 0.0, ""]
+      [JSON.Token.Number, 0, 5, 0, 12300, ""]
```

Comment:
This is not related to your PR, but when I changed floating-point printing (in the latest two Savi releases) it now always includes a trailing `.0` for floating-point numbers, just to help disambiguate them visually from integers.

```suggestion
      [JSON.Token.Number, 0, 5, 0, 12300.0, ""]
```